apiary.apib

```diff
@@ -3673,6 +3673,23 @@ Update a draft work order.
 
 + Response 204
 
+### workOrders.attachImage [POST /workOrders.attachImage]
+
+Attach image to a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+        + image: `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAAAAXNSR0IArs4c6QAAAVlpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8AAAASUVORK5CYII=` (string, required) - PNG image, base64 encoded
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + type: `workOrderImage` (string)
```

Comment:
Q: the type is not 100% decided yet, right?

Reply:
Not sure, that's what was in the original work order proposal

Reply:
We discussed this yesterday and thought it should just remain a `fileId` just to avoid having to convert between WO and File UUIDs everywhere.

Only now got to updating the proposal. Sorry about that.

Reply:
No worries, I somehow missed that decision